src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,20 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        enum ColorBase{
+            BLUE,
+            YELLOW,
+            GREEN,
+            RED;
+        }
+
+        Random random = new Random();
+        int randomNum = random.nextInt(ColorBase.values().length);
+        ColorBase[] randomColor = ColorBase.values();
+
+        return randomColor[randomNum].toString();
```

Comment:
Instead of using `toString()`, you should use the `name()` method to get the String representation of the enum constants. This ensures that the String representation is consistent and not affected by any potential overrides of the `toString()` method.